packages/arb-avm-cpp/data_storage/src/datastorage.cpp

```diff
@@ -52,17 +53,19 @@ DataStorage::DataStorage(const std::string& db_path) {
     assert(status.ok());
     txn_db = std::unique_ptr<rocksdb::TransactionDB>(db);
     default_column = std::unique_ptr<rocksdb::ColumnFamilyHandle>(handles[0]);
-    blocks_column = std::unique_ptr<rocksdb::ColumnFamilyHandle>(handles[1]);
-    node_column = std::unique_ptr<rocksdb::ColumnFamilyHandle>(handles[2]);
+    core_column = std::unique_ptr<rocksdb::ColumnFamilyHandle>(handles[1]);
```

Comment:
Really just a note for the future, but we should make sure to clean up any excess columns we're creating here if we end up not using them